### PR TITLE
Add detection for `gap` CSS property for flexbox

### PR DESCRIPTION
**Pull request by chris13524**
This PR implements detection for gaps in flexboxes. https://caniuse.com/#feat=flexbox-gap

### CODE REVIEW

File: feature-detects/css/flexgap.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  "notes": [{
+    "name": "W3C Spec",
+    "href": "https://www.w3.org/TR/css-align-3/#gaps"
+  }]
+}
+!*/
+define(['Modernizr'], function(Modernizr) {
+  Modernizr.addTest('flexgap', function() {
+  	// create flex container with row-gap set
+  	var flex = document.createElement('div');
```
Comment (rejas) [inline feature-detects/css/flexgap.js]: Please use the method `createElement` for adding an element to the dom

**Comment by chris13524** [comment]
@rejas haha sorry about that. First time pushing, didn't read contribution wiki. Should all be fixed now. Thanks!

File: feature-detects/css/flexgap.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/*!
+{
+  "name": "Flex Gap",
+  "property": "flexgap",
+  "caniuse": "flexbox-gap",
+  "tags": ["css", "flexbox"],
+  "notes": [{
+    "name": "W3C Spec",
+    "href": "https://www.w3.org/TR/css-align-3/#gaps"
+  }]
```
Comment (rejas) [inline feature-detects/css/flexgap.js]: you can also add yourself as an author here

Comment (chris13524) [reply feature-detects/css/flexgap.js]: Thanks!

File: feature-detects/css/flexgap.js [hunk 3]
```diff
@@ -7,26 +7,27 @@
   "notes": [{
     "name": "W3C Spec",
     "href": "https://www.w3.org/TR/css-align-3/#gaps"
-  }]
+  }],
+  "authors": "Chris Smith (@chris13524)"
```
Comment (rejas) [inline feature-detects/css/flexgap.js]: Authors has to be an array. That would have shown up on the travis tests if there wasnt a bug with my code :-) Fixed that so if you rebase now against master you wil lsee that the tests fail :-)

Comment (chris13524) [reply feature-detects/css/flexgap.js]: D'oh!

**Comment by rejas** [comment]
Thx :-)

**Pull request merged**